docs/superdesk-ninjs-schema.json

```diff
@@ -15,11 +15,11 @@
 			"type" : "string"
   		}
 	},
+	"required": ["guid"],
```

Comment:
Do we need to change this line as well https://github.com/superdesk/superdesk-core/blob/master/docs/superdesk-ninjs-schema.json#L2 ?

Reply:
That contains the description of the keywords used in the schema. We added some custom keywords so we may create our own document but I think this should be done in another task.

Reply:
that's fine if you want to do in a separate task.